samples/namespacescope-sample/readme.txt

```diff
@@ -0,0 +1,5 @@
+This sample illustrates Namespace scoped application relevant changes in the manifest.
+so it has following changes compared to the azure vote sample

```

Comment:
Should we include below link 

[https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-extensions#extension-scope](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-extensions#extension-scope)